lib/demo_web/controllers/user_auth.ex

```diff
@@ -0,0 +1,139 @@
+defmodule DemoWeb.UserAuth do
```

Comment:
Should the module be called user_auth_controller for consistency with `DemoWeb.UserConfirmationController` etc

Reply:
It is not a controller. :) But helpers used by the controllers.

Reply:
ah I see! make sense

Reply:
maybe auth_helper then? I was thrown off on that too, I'm trying to move from laravel to phoenix for a reddit/slack/talkyard/stackoverflow mishmash. 

I like how laravel does auth, and organizes by sub folder, so I'm doing that in my own version, as a newbie, I don't want to step on anyone's toes. I've used devise a LOT in pet projects not production as I haven't put any rails into production. 

I think the naming conventions like this seem a little better: 

Auth.AuthHelper => lib/demo_web/controllers/auth/auth_helper.ex
Auth.ConfirmationController => .../confirmation_controller.ex
Auth.RegistrationController => .../registration_controller.ex
Auth.ResetPasswordController => .../reset_password_controller.ex
Auth.SessionController => .../session_controller.ex
etc... 

Just a suggestion, from someone w/out a lot of experience in the framework, but this jives w/ my 8 years using laravel brain.